14_2_Using_i2p.md

```diff
@@ -0,0 +1,50 @@
+# Chapter 14.2: Using i2p
+
+The Invisible Internet Project (I2P) is a fully encrypted private network layer. It uses a distributed [network database](https://geti2p.net/en/docs/how/network-database) and encrypted unidirectional tunnels between your and your peers.
+
+Basic differences between Tor and i2p: https://geti2p.net/en/comparison/tor
```

Comment:
We'd prefer to see a list of the most important differences (in your own words), and then link to that for more.

Reply:
Still researching about differences and discussing with few people. I agree it will be better if I mention few differences and provide links for details.

Reply:
Done. Added a table in https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/commit/e6faf1cc07f4477c6508bbf82b1dc59940f101d9